opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/TaskRegisterFragmentPresenter.java

```diff
@@ -87,13 +91,10 @@ public void searchGlobally(String uniqueId) {
         //do nothing, tasks not searchable globally
     }
 
-    public String getMainCondition() {
+    public Pair<String, String[]> getMainCondition() {
```

Comment:
Why not implement this in the `TaskRegisterFragment` where it's actually an abstract method that needs implementation?